src/RoseBikesSpryker/Zed/ChannableApi/Business/Hydrator/Plugin/AbstractHydrator.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace RoseBikesSpryker\Zed\ChannableApi\Business\Hydrator\Plugin;
+
+class AbstractHydrator
+{
+    /**
+     * @const int
+     */
+    public const PRICE_MULTIPLIER = 100;
+
+    /**
+     * @const int
+     */
+    public const PRICE_SCALE = 2;
+
+    /**
+     * @param string $price
+     *
+     * @return int
+     */
+    public function convertPriceToInteger(string $price): int
+    {
+        return (int)bcmul($price, self::PRICE_MULTIPLIER, self::PRICE_SCALE);
```

Comment:
```suggestion
        return (int)bcmul($price, static::PRICE_MULTIPLIER, static::PRICE_SCALE);
```

so ließe sich das auf projektebene überschreiben